.env.example

```diff
@@ -1,8 +1,7 @@
 DEBUG=True
 DJANGO_SECRET_KEY=3YSxRH/mcqSJpkk40s+0qy+uh+9o5LIBdZtwHOo6oNaLe8+Mx6
-DATABASE_URL=postgresql://postgres:postgres@localhost:5432/antiplag
-CELERY_BROKER_URL=pyamqp://
-CELERY_RESULT_BACKEND=rpc://
+DATABASE_URL=postgresql://postgres:postgres@db:5432/antiplag
+CELERY_BROKER_URL=pyamqp://broker
+CELERY_RESULT_BACKEND=rpc://broker
 CELERY_RESULT_PERSISTENT=True
-ELASTIC_HOST=elastic:pNA4gPABp6p2TMnowIcN@elastic.antiplag.sk:80
-TESSERACT_PATH=
+ELASTIC_HOST=user:password@elastic.antiplag.sk:80
```

Comment:
Do you mean `ELASTIC_HOST`? I would leave it as is, since it is an example environment setup and should guide user on what to do. 

Does it matter what `DJANGO_SECRET_KEY` we use in development?

Reply:
Well - let's think about two cases.

In case that example has empty value, e.g. `ELASTIC_HOST=`, the developer after copy-pasting the `.env.example` to `.env` file would get an error message saying that this variable is not defined. In case it would be prefilled like in this case, the error would not be raised on build, but on runtime. As we're connecting to elastic asynchronously in the worker, HTTP responses wouldn't contain this type of error that the connection wasn't successful due to the incorrect credentials. The logs would be then harder to debug for less skilled developers.

The same applies to `DJANGO_SECRET_KEY`, which could be easily forgotten to be changed when deploying to the server - which could bring potential breach as this salt was exposed on github before. On the other hand the absence of value would be more straightforward.

What do you think?

Reply:
1. Both errors will be raised on runtime - variable not defined on server/worker startup and connection error during connection attempt. 
I get what you mean, but if you copy-paste without checking what you are doing, it is your fault. It is even explicitly stated in readme that you need to specify correct credentials. Error should be easily seen in worker logs. 
-> Your call

2. Anything is possible, but if we keep secret key from `.env.example`, we might another, bigger issues with our deployment..


What about `CELERY_BROKER_URL`? What if user has another broker and copy-pastes example env? Should we keep it blank as well?

Reply:
1. What I'm saying it is doing it error-proof. Let's try to be more empathic to less skilled developers. If you cannot run the server, you try to solve stuff using server logs. Even if you ran docker in detached mode, you don't need console to see the error in the http response. The celery, on the other hand, is running in backround and the errors could only be seen in the console.

I mean, having blank variables versus incorrectly initialized variables - I would pick the blank option. 

2. Having it blank we avoid the problem.

3. `CELERY_BROKER_URL` shouldn't be actually in .env file in my opinion - it is hardcoded value, which does not depend on environment, on which is the server running on. As it's the containers ip adress, it's the same for everyone.